<ui version="4.0" >
 <class>ThermalRadiationAdvancedDialogForm</class>
 <widget class="QDialog" name="ThermalRadiationAdvancedDialogForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelFreq" >
        <property name="text" >
         <string>Iteration resolution frequency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" colspan="2" >
       <widget class="QLineEdit" name="lineEditFreq" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Code_Saturne key words: NREFQR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="labelTSRay" >
        <property name="text" >
         <string>Radiative source term calculus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="6" >
       <widget class="QComboBox" name="comboBoxTSRay" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Code_Saturne key words: IDIVER</string>
        </property>
        <item>
         <property name="text" >
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QLabel" name="labelPrintT" >
        <property name="text" >
         <string>Verbosity level for wall temperature</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="6" >
       <widget class="QComboBox" name="comboBoxPrintT" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Code_Saturne key words: IIMPAR</string>
        </property>
        <item>
         <property name="text" >
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="4" >
       <widget class="QLabel" name="labelPrintL" >
        <property name="text" >
         <string>Verbosity level for brigthness resolution</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="6" >
       <widget class="QComboBox" name="comboBoxPrintL" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Code_Saturne key words: IIMLUM</string>
        </property>
        <item>
         <property name="text" >
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>280</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ThermalRadiationAdvancedDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ThermalRadiationAdvancedDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
